linkerd/proxy/tap/src/accept.rs

```diff
@@ -58,9 +58,9 @@ impl AcceptPermittedClients {
 }
 
 impl Service<Connection> for AcceptPermittedClients {
-    type Response = ();
+    type Response = Box<dyn Future<Item = (), Error = Error> + Send + 'static>;
     type Error = Error;
-    type Future = ServeFuture;
+    type Future = Box<dyn Future<Item = Self::Response, Error = Self::Error> + Send + 'static>;
```

Comment:
This is always a `FutureResult`, so it doesn't need to be boxed:
```suggestion
    type Future = future::FutureResult<Self::Response, Self::Error>>;
```